common/config/debug_dump.nsh

```diff
@@ -30,47 +30,60 @@ for %m in 0 1 2 3 4 5 6 7 8 9 A B C D E F then
     if exist FS%m:\acs_results then
         FS%m:
         cd FS%m:\acs_results
-        if exist uefi_dump then
-            echo "UEFI debug logs already run"
-            echo "press any key to rerun UEFI debug logs"
-            FS%m:\EFI\BOOT\bbr\SCT\stallforkey.efi 10
-            if %lasterror% == 0 then
-                goto DEBUG_DUMP
-            else
-                goto Done
-            endif
+    endif      
```

Comment:
Hi Sunny, this will break the ACS IR UEFI debug dump log collection. The uefi_dump folder might be created in the boot partition instead of the results partition.

Reply:
@ownia I made a fix. Could you give it a try?
[debug_dump.zip](https://github.com/user-attachments/files/15825557/debug_dump.zip)

Reply:
Hi @sunnywang-arm,
For an IR generic automation test, UEFI debug dump process is after the capsule update. So the current implementation will not return to the `acs_results` folder and will lead the uefi_dump folder to be generated under the `\acs_results\app_output\`, which is not what we expected.
I think your attached patch can fix it.